### PR TITLE
cfengine 3.12.2 sha256 update

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

~Package seems not did published to `https://cfengine-package-repos.s3.amazonaws.com/tarballs/cfengine-3.12.2-2.tar.gz`?~ (released with the same artifact URL, so I updated the sha256 values accordingly)

### CODE REVIEW

**Comment by chenrui333** [comment]
```
01:33:19 ==> Downloading https://github.com/cfengine/core/archive/3.12.2-2.tar.gz
01:33:19 Already downloaded: /Users/brew/Library/Caches/Homebrew/downloads/e4d329be754003d7c5b158d8d17abfc738e28310f96436e8b5d3e78a5226429d--core-3.12.2-2.tar.gz
01:33:19 ==> Verifying e4d329be754003d7c5b158d8d17abfc738e28310f96436e8b5d3e78a5226429d--core-3.12.2-2.tar.gz checksum
01:33:19 tar xof /Users/brew/Library/Caches/Homebrew/downloads/e4d329be754003d7c5b158d8d17abfc738e28310f96436e8b5d3e78a5226429d--core-3.12.2-2.tar.gz -C /private/tmp/d20190531-42247-9q22qh
01:33:19 cp -pR /private/tmp/d20190531-42247-9q22qh/core-3.12.2-2/. /private/tmp/cfengine-20190531-42247-1b57pz7/core-3.12.2-2
01:33:19 chmod -Rf +w /private/tmp/d20190531-42247-9q22qh
01:33:19 ==> ./configure --disable-dependency-tracking --prefix=/usr/local/Cellar/cfengine/2 --with-workdir=/usr/local/var/cfengine --with-lmdb=/usr/local/opt/lmdb --with-pcre=/usr/local/opt/pcre --without-mysql --without-postgresql
01:33:19 Failed to execute: ./configure
01:33:19 
01:33:19 ==> Formula
01:33:19 Path: /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/cfengine.rb
```

**Comment by chenrui333** [comment]
01:33:19 ==> ./configure --disable-dependency-tracking --prefix=/usr/local/Cellar/cfengine/2 --with-workdir=/usr/local/var/cfengine --with-lmdb=/usr/local/opt/lmdb --with-pcre=/usr/local/opt/pcre --without-mysql --without-postgresql
01:33:19 Failed to execute: ./configure

File: Formula/cfengine.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Cfengine < Formula
   desc "Help manage and understand IT infrastructure"
   homepage "https://cfengine.com/"
-  url "https://cfengine-package-repos.s3.amazonaws.com/tarballs/cfengine-3.12.2.tar.gz"
```
Comment (chenrui333) [inline Formula/cfengine.rb]: The package is not available in s3 bucket, reported in upstream repo.

Comment (vpodzime) [reply Formula/cfengine.rb]: > The package is not available in s3 bucket, reported in upstream repo.

The tarball was replaced by the re-released version. So the URL is still just https://cfengine-package-repos.s3.amazonaws.com/tarballs/cfengine-3.12.2.tar.gz (no `-2`).

Comment (chenrui333) [reply Formula/cfengine.rb]: Thanks for confirming it, I have just updated the sha256 values for the two tarball files.

**Comment by chenrui333** [comment]
Close in favor of https://github.com/Homebrew/homebrew-core/pull/41533

**Pull request closed without merging**